darshan-util/pydarshan/tests/input/__init__.py

```diff
@@ -6,5 +6,6 @@ def installed_path(filename):
 
 test_data_files_dxt = {
     "sample-dxt-simple.darshan": installed_path("sample-dxt-simple.darshan"),
+    "sample-badost.darshan": installed_path("sample-badost.darshan")
```

Comment:
Why would we add a file without `DXT` records to a test dictionary of files that contain `DXT` records, as both the `test_data_files_dxt` name and its usage in the code implies?

Reply:
(there are better ways to add files to the benchmark suite for example, that don't involve masquerading as a DXT containing file)

Reply:
That's a good point. I could create a new dictionary. It will complicate the setup logic a little but that's not a big deal.

Reply:
The latest commit splits the newly added log into its own data file dictionary.